vms/evm/acp226/acp226.go

```diff
@@ -21,6 +21,10 @@ const (
 	// MaxDelayExcessDiff (Q) is the maximum change in excess per update
 	MaxDelayExcessDiff = 200
 
+	// DefaultDelayExcess represents the initial (≈2000ms) delay excess.
+	// Formula: ConversionRate (2^20) * ln(2000) + 1
+	DefaultDelayExcess = 7_970_124
```

Comment:
I'm wondering if this would be more descriptive as the `InitialDelayExcess`. If this is limited to the C-chain, I think it makes more sense as `Initial`, but if there is a situation with subnet-evm that this makes more sense as `Default` then this lgtm.

Reply:
I think we would also want this to be subnet-evm default. Otherwise we should really move them to each repo (see this comment https://github.com/ava-labs/avalanchego/pull/4292#discussion_r2362642275)